docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -103,6 +103,19 @@ For example:
 The same applies to `slotProps` and `componentsProps`.
 :::
 
+### ✅ Rename slots types
+
+The slot interfaces got renamed to match with `@mui/base` naming convention.
```

Comment:
@LukasTy @alexfauquette do you think we should add the full list (in a collapsed node clearly) or link to the codemod file where people can find the full list?

Reply:
I agree with Alex on this one. 🤔 
However, I wouldn't mind having the codemod linked here. 👍